src/views/RootView.js

```diff
@@ -20,7 +22,7 @@ export function RootView(props) {
         <NetworkErrorContainer />
       </section>
 
-      <section className="menu__section" onClick={() => resetGrid()}>
+      <section className="menu__section" onClick={() => null}>
```

Comment:
we have a noop (no-operation function) for `onClick` here, may not need to define it at all here